Doc/library/os.rst

```diff
@@ -4170,7 +4170,7 @@ to be ignored.
    The "l" and "v" variants of the :func:`exec\* <execl>` functions differ in how
    command-line arguments are passed.  The "l" variants are perhaps the easiest
    to work with if the number of parameters is fixed when the code is written; the
-   individual parameters simply become additional parameters to the :func:`execl\*`
+   individual parameters simply become additional parameters to the :func:`execl\* <execl>`
```

Comment:
This just links to the entry we're already in; maybe it would be better to just suppress the link?

```suggestion
   individual parameters simply become additional parameters to the :func:`!execl\*`
```